utils/strings/capitalize.js

```diff
@@ -0,0 +1,2 @@
+// capitalize value to Value
+module.exports = (value) => value[0].toUpperCase() + value.slice(1)
```

Comment:
Weird idea, but does that work with short strings like `a`, or would `.slice(1)` return `undefined`? Maybe we could also use the `${...}${...}` format (https://eslint.org/docs/latest/rules/prefer-template).

Reply:
this works: `capitalize("a") = A`, added to tests